### PR TITLE
Dockerfile for CGE's mlst tool and their mlst database.

### PR DESCRIPTION
**Pull request by mokrobial**
Container for CGE's mlst tool and database.

### CODE REVIEW

**Comment by kapsakcj** [comment]
Hi @mokrobial - thanks for the PR! I appreciate you contributing to this repository. I didn't even realize that CGE had their own MLST tool, but it make sense - they have a tool for just about everything 😄 

The dockerfile builds with no issues, but I would like to suggest some changes to improve things before we merge the PR & build the image on dockerhub.

The main changes I'd like to see include pinning the versions of `mlst`, it's associated database, and `kma`.

`git clone`'ing the bitbucket repos works, but the docker image will be different if we build it today and again in 6 months using the same dockerfile. Part of the goal of this project is to have the containers stay _as static as possible_ so that means downloading code from a GitHub release, or in the case of bitbucket, downloading the code from a specific tag.

Check out the SerotypeFinder 2.0.1 dockerfile for some code you could grab and/or use as an example: https://github.com/StaPH-B/docker-builds/blob/master/serotypefinder/2.0.1/Dockerfile

File: CGE-mlst/Dockerfile [hunk 1]
```diff
@@ -0,0 +1,61 @@
+FROM debian:stretch
+
+ENV DEBIAN_FRONTEND noninteractive
+
+# metadata
+LABEL base.image="ubuntu:xenial"
+LABEL dockerfile.version="1"
+LABEL software="mlst"
+LABEL software.version="2.19.0"
```
Comment (kapsakcj) [inline CGE-mlst/Dockerfile:9]: I think this version should be 2.0.4? That's the latest tag I can see on the bitbucket repo

File: CGE-mlst/Dockerfile [hunk 2]
```diff
@@ -0,0 +1,61 @@
...
+LABEL description="CGE Multi-Locus Sequence Typing 2.0"
+LABEL website="https://cge.cbs.dtu.dk/services/MLST/"
```
Comment (kapsakcj) [inline CGE-mlst/Dockerfile:11]: could you also add a label for the bitbucket repo? `LABEL code.repository="https://bitbucket.org/genomicepidemiology/mlst/src/master/"`

File: CGE-mlst/Dockerfile [hunk 3]
```diff
@@ -0,0 +1,61 @@
...
+LABEL maintainer="Center for Genomic Epidemiology"
```
Comment (kapsakcj) [inline CGE-mlst/Dockerfile:12]: please change these labels to your name and email address

**Comment by kapsakcj** [comment]
also might be good to make a comment in the dockerfile that says that the blast version is 2.6.0 and that bedtools is v2.26.0
```
# blastn -version
blastn: 2.6.0+
 Package: blast 2.6.0, build Jan 15 2017 17:12:27

# bedtools --version
bedtools v2.26.0
```

**Comment by mokrobial** [comment]
Hi, 

Thank you SO much for the feedback! This was my first container and I really appreciate the detailed feedback. I’ll edit and try again :)

mk

**Comment by kapsakcj** [comment]
You're very welcome! Let me know if you have questions about anything.

**Comment by erinyoung** [comment]
@mokrobial ,

Are there any updates on this?

I haven't seen any changes since @kapsakcj 's comments.

**Comment by erinyoung** [comment]
Thank you for contributing! If you continue to work on this, please reopen the pull request.

**Pull request closed without merging**